QUARKUS-1414.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-1414 Service Binding improvements
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1414
+
+The goal of this feature is to improve the provided technology preview support of OpenShift Service Binding in Quarkus
+applications.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* We need to ensure all the supported means of producing Quarkus deployments in OpenShift work with service binding
+  * Source S2I
+  * Docker build
+  * Binary S2I
+* Orchestration of OSD will have to be automated to ensure easy testing on OSD with RHOSAK service bindings. This, as of
+  now, is done manually (see QUARKUS-1027).
+* New tests have to be implemented in Quarkus QE test suite, with applications utilizing the following service bindings:
+  * RHOSAK
+  * Apicurio
+  * MongoDB
+  * MySQL
+  * PostgreSQL
+
```

Comment:
Do we need to cover all of the above services in RHBQ testing.

I would be concerned primarily about RHOSAK / Kafka.

Reply:
The original messaging by PM was that the unit testing is sufficient, which makes me considerably uncomfortable - but considering this is TP, there's no priorities, probably

Reply:
IMHO RHOSAK is the main one, because it's priority for Middleware